### PR TITLE
Josh branch page

### PR DESCRIPTION
**Pull request by KnightJam1**
Add the medical contributions page

### CODE REVIEW

**Comment by tameTNT** [comment]
I can't wait to try out whatever beautiful creation you've just added here...

**Comment by KnightJam1** [comment]
I dare you to press the "Do Not Press" button

**Review by tameTNT — CHANGES_REQUESTED** [review]
Mermermer he's requested changes *again*. This guy is never happy...

File: medical.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+
+        <script>
+            function play() {
+                var audio = document.getElementById("audio");
+                audio.play();
+                document.getElementById('explode').style.display='block';
+                document.getElementById('dontButton').style.display='none';
+            }
+            audio.addEventListener("ended", function() {
```
[the remark this answers is not in the document]
Comment (KnightJam1) [reply medical.html:80]: No idea how you got that, I removed it a while ago??? Well, it's gone now.

[the remark this answers is not in the document]
Comment (tameTNT) [reply medical.html]: Probably like a forgot to fetch/push thing or smth idk? 🤷

**Pull request merged**